src/components/external-link/external-link.css

```diff
@@ -0,0 +1,6 @@
+[class^="icon"] {
+  width: 12px;
+  height: 12px;
+  position: relative;
+  top: 1.5px;
+}
```

Comment:
This CSS affects all elements in the app that have a class starting with the string `icon` (which is definitely not what we want, that gets a little funky). Try rebasing your branch again and re-`yarn`ing - it looks like the changes you made to `stripes-components` might not have made it here. This CSS file shouldn't be needed.

<img width="311" alt="screen shot 2018-07-19 at 7 40 31 pm" src="https://user-images.githubusercontent.com/230597/42977230-9fa1c8c2-8b8b-11e8-923b-283663b9c1ac.png">